lib/bootstrap/bundler.rb

```diff
@@ -196,6 +196,44 @@ def debug?
       ENV["DEBUG"]
     end
 
+    # @param plugin_names [Array] logstash plugin names that are going to update
+    # @return [Array] gem names that plugins depend on, including logstash plugins
+    def consolidate_dependencies(plugin_names, with_plugin_name = true)
```

Comment:
Naming: "consolidate" doesn't really tell me what to expect.

Maybe "expand_safe_dependencies"? Something that tells us we are going to do _more_ than the minimum.

Nit: If the caller has enough info to pass us `false` when it doesn't want the original plugin names, or if it knows to avoid passing a second argument if it _does_ want them in the list, then it should be responsible for adding them, not this method. Let's move that responsibility to the caller to simplify the responsibilities of this method.

Reply:
I change the name to `expand_logstash_mixin_dependencies`